ocean-client/addons/command-center/src/commands/command.ts

```diff
@@ -87,28 +110,29 @@ export abstract class Command {
     return undefined
   }
 
-  protected isBasicCommand(): boolean {
+  protected isInfoCommand(): boolean {
     return false
   }
 
   abstract doExecution(): Promise<unknown>
   abstract availableFor(): Bot[]
 
   async execute(): Promise<unknown> {
-    this.parseBot()
+    // possibility to be a chained command, therefore bot is set before execution
+    // no need and no possibility to parse bot again
+    if (!this.bot) {
+      this.doGeneralBotAvailabilityChecks()
+      this.parseBot()
```

Comment:
first parse, then do checks. otherwise the "isUndecided" in the checks is "pointless"